ydb/core/base/table_index.cpp

```diff
@@ -28,8 +28,22 @@ bool Contains(const auto& names, std::string_view str) {
     return std::find(std::begin(names), std::end(names), str) != std::end(names);
 }
 
-constexpr std::string_view ImplTables[] = {
-    ImplTable, NTableVectorKmeansTreeIndex::LevelTable, NTableVectorKmeansTreeIndex::PostingTable,
+bool ContainsSystemColumn(const auto& columns) {
+    for (const auto& column : columns) {
+        if (column.StartsWith(SYSTEM_COLUMN_PREFIX)) {
+            return true;
+        }
+    }
+    return false;
+}
+
+const TString ImplTables[] = {
+    ImplTable,
+    NTableVectorKmeansTreeIndex::LevelTable,
+    NTableVectorKmeansTreeIndex::PostingTable,
+    NTableVectorKmeansTreeIndex::PrefixTable,
+    TString{NTableVectorKmeansTreeIndex::PostingTable} + NTableVectorKmeansTreeIndex::BuildSuffix0,
+    TString{NTableVectorKmeansTreeIndex::PostingTable} + NTableVectorKmeansTreeIndex::BuildSuffix1,
```

Comment:
I'm not really sure is it needed or not, but I find place where we check that impl table one of this or crash.
So it's my paranoia